### PR TITLE
[WIP] Upgrade to Ecto 3.0

### PR DESCRIPTION
**Pull request by nathany**
work in progress. I'm looking into the test failures

### CODE REVIEW

File: mix.exs [hunk 1]
```diff
@@ -46,8 +46,9 @@ defmodule Authable.Mixfile do
   # Type "mix help deps" for more examples and options
   defp deps do
     [
-      {:postgrex, "~> 0.13.5"},
-      {:ecto, "~> 2.2.10"},
+      {:postgrex, "~> 0.14.0"},
+      {:ecto, "~> 3.0.0", override: true},
```
Comment (nathany) [inline mix.exs]: This override won't be necessary after ex_machina is updated.

See https://github.com/thoughtbot/ex_machina/pull/301

**Comment by nathany** [comment]
I'm not sure if this is related to the test failures. I'm asking in Slack.

> A downside of this approach is that migrations cannot run dynamically during test under the Ecto.Adapters.SQL.Sandbox, as the sandbox is unable to share a single connection across processes to guarantee the changes can be reverted. https://github.com/elixir-ecto/ecto/blob/master/CHANGELOG.md

**Comment by tpitale** [comment]
@nathany Looks like ex_machina Ecto 3 PR has been merged, and appears to be released: https://hex.pm/packages/ex_machina

File: mix.exs (hunk 1 above)
Comment (tpitale) [reply mix.exs]: Looks like ex_machina Ecto 3 PR has been merged, and appears to be released: https://hex.pm/packages/ex_machina

**Comment by mustafaturan** [comment]
Closed in favor of https://github.com/mustafaturan/authable/pull/116

**Pull request closed without merging**